pktoken/pktoken.go

```diff
@@ -357,26 +384,18 @@ func (p *PKToken) VerifyCosSig(cosPk jwk.Key, alg jwa.KeyAlgorithm) error {
 	return nil
 }
 
-func (p *PKToken) Verify(msg string, sig []byte) error {
-	jwsSig, err := jws.Parse(sig)
+func (p *PKToken) Hash() (string, error) {
+	// get pktoken bytes
+	pktJson, err := p.ToJSON()
 	if err != nil {
-		return err
-	}
-
-	if msg != string(jwsSig.Payload()) {
-		return fmt.Errorf("Message does not match signed message")
-	}
-
-	alg, _, upk, err := p.GetCicValues()
-	if err != nil {
-		return err
+		return "", err
 	}
 
-	_, err = jws.Verify(sig, jws.WithKey(alg, upk))
+	hasher := sha256.New()
+	_, err = hasher.Write(pktJson)
 	if err != nil {
-		return err
+		return "", err
 	}
-
-	// verified
-	return nil
+	hash := hasher.Sum(nil)
+	return string(util.Base64EncodeForJWT(hash)), nil
```

Comment:
hexstring rather than base64. Users can then compare the values without deserializing.

Reply:
I think you made this same comment on my previous PR, but the conclusion was that you can do so with either.

Reply:
DIdn't you end up doing hex tho? We should do hex (all caps) to signal "do not deserialize,  no highly esteemed deed is commemorated here,  this string is best shunned and left unserialized"